packages/migrate/index.js

```diff
@@ -4,6 +4,7 @@ const glob = require("glob");
 const expect = require("@truffle/expect");
 const Config = require("@truffle/config");
 const Migration = require("./Migration");
+const emitEvent = require("./emitEvent");
```

Comment:
Could we change this to like `makeEmit`? Something like:
```typescript
const emit = makeEmit(options);
emit("migrate:dryRun:notAccepted");
```

What do you think?

Reply:
Is the purpose of this that you don't want to pass `options` each time?

Reply:
Hmm but this is kind of annoying because on the one hand, `Migration` is a class and we can set this new thing `this.emit`. `Migrate` is just an object with utilities in; there is no instantiation so we'd have to create a new one in each context in which it is used, which sucks. Did you have a different kind of idea?

Reply:
This is probably fine (and not worth blocking merge over)